wire-ios-notification-engine/Sources/NotificationSession.swift

```diff
@@ -137,7 +137,19 @@ public class NotificationSession {
             applicationContainer: sharedContainerURL
         )
 
+        guard coreDataStack.storesExists else {
+            throw InitializationError.coreDataMissingSharedContainer
+        }
+
+        guard !coreDataStack.needsMigration  else {
+            throw InitializationError.coreDataMigrationRequired
```

Comment:
that might be a feature actually to let the user know with a dummy notification to open his app in order to do the migration

Reply:
I'll keep it in mind for the follow up! thanks